ChangeLog

```diff
@@ -493,6 +500,17 @@ API Changes
      and mbedtls_hmac_drbg_update() respectively, but the new functions
      report errors whereas the old functions return void. We recommend that
      applications use the new functions.
+   * The following functions in the random generator modules have been
```

Comment:
These unrelated additions in the changelog don't belong here.

Reply:
Oops. Thanks for catching that. I'll fix it.